nncf/tensorflow/graph/metatypes/keras_layers.py

```diff
@@ -83,6 +84,16 @@ class TFLayerWithWeightsMetatype(TFLayerMetatype):
     bias_attr_name = None  # type: Optional[str]
 
 
+@KERAS_LAYER_METATYPES.register()
+@UNKNOWN_METATYPES.register()
+class TFLayerUnknownMetatype(TFLayerMetatype):
+    name = 'unknown'
+
+    @classmethod
+    def get_all_aliases(cls) -> List[str]:
+        return [cls.name]
```

Comment:
This is unnecessary, reuse a common `UnknownMetatype` for all FWs

Reply:
Done